src/test/data/JsonSerializableAddressBookTest/typicalProjectsAddressBook.json

```diff
@@ -35,5 +35,27 @@
     "budget" : "9482442",
     "deadline" : "2021-10-05",
     "tagged" : [ ]
-  } ]
+  } ],
+  "tasks" : [{
+    "taskDescription" : "Todo 2101",
+    "taskDeadline": "2022-01-11"
+  }, {
+    "taskDescription" : "Todo 2102",
+    "taskDeadline": "2022-02-16"
+  }, {
+    "taskDescription" : "Todo 2103",
+    "taskDeadline": "2022-03-15"
+  }, {
+    "taskDescription" : "Todo 2104",
+    "taskDeadline": "2022-04-14"
+  }, {
+    "taskDescription" : "Todo 2105",
+    "taskDeadline": "2022-05-13"
+  }, {
+    "taskDescription" : "Todo 2106",
+    "taskDeadline": "2022-06-12"
+  }, {
+    "taskDescription" : "Todo 2107",
+    "taskDeadline": "2022-07-11"
+  }]
```

Comment:
Should shuffle the data here a bit. This set of data does not allow us to test for sort by deadline properly.